spring-integration-test-support/src/main/java/org/springframework/integration/test/util/TestUtils.java

```diff
@@ -85,6 +105,11 @@ else if (i == tokens.length - 1) {
 		return (T) value;
 	}
 
+	/**
+	 * Create a {@link TestApplicationContext} instance
+	 * supplied with the besic Spring Integration infrastructure.
```

Comment:
Welcome back, @Toparvion !

M-m-m. I think you mean the typo: s/besic/basic.

Otherwise you have just copied/pasted text 

Maybe that was intention to pull my attention, but since I did a mistake it isn't obvious for me where I'm wrong 😄